System/fan-speed.5s.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+# <bitbar.title>Fan Speed</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Eric Ripa</bitbar.author>
+# <bitbar.author.github>eripa</bitbar.author.github>
+# <bitbar.desc>This plugin displays the current fan speed from SMC readings (requires external dependencie)</bitbar.desc>
+# <bitbar.image></bitbar.image>
+# <bitbar.dependencies>smc binary http://www.eidac.de/smcfancontrol/smcfancontrol_2_4.zip</bitbar.dependencies>
+
+declare -a FANS # list available fans with smc -f
+declare -a FAN_LABEL # list available fans with smc -f
+FANS=("Fan #0" "Fan #1" "Fan #2")
+# FAN_LABEL=("CPU" "ODD" "HDD") # Uncomment to add label, must 1-to-1 map the above array
+FAN_SPEEDS="♨︎ " # Set your own prefix
+
+for ((i = 0; i < ${#FANS[@]}; i++)) ; do
+  FAN_SPEED=$(/usr/local/bin/smc -f | grep -A1 "${FANS[$i]}" | grep Actual | awk '{printf "%s\n", $4}')
+  if [ "$FAN_SPEED" != "" ] ; then
+    if [ ! -z ${FAN_LABEL+x} ]; then
```

Comment:
This var needs double quoting according to [shellcheck](http://www.shellcheck.net/)